front/src/applications/rollingStockEditor/components/RollingStockEditorCurves.tsx

```diff
@@ -38,10 +38,10 @@ export default function RollingStockEditorCurves(props: {
         }
       | undefined
     )[][] =
-      speedsList !== undefined && effortsList !== undefined
+      speedsList && effortsList
         ? effortsList.map((effort, index) => [
-            { value: (speedsList[index] * 3.6).toString() },
-            { value: (effort / 10).toString() },
+            { value: Math.round(speedsList[index] * 3.6).toString() },
+            { value: Math.round(effort / 10).toString() },
```

Comment:
Is it rounded at unit here ? Can we allow 2 digits after comma ?

Reply:
Currently, it's rounded at unit, but I can allow 2 digits after comma sure :)

Reply:
I edited my message, i actually don't think it's really necessay to have 2 digits after comma, you can leave it like that :)